crates/rome_control_flow/src/lib.rs

```diff
@@ -29,6 +29,16 @@ impl<L: Language> ControlFlowGraph<L> {
             node,
         }
     }
+
+    pub fn get(&self, id: BlockId) -> &BasicBlock<L> {
+        // SAFETY: safe conversion because a block id corresponds to its index in `self.blocks`.
+        let block_index = id.index() as usize;
+        &self.blocks[block_index]
+    }
+
+    pub fn block_ids(&self) -> impl Iterator<Item = (BlockId, &BasicBlock<L>)> {
+        (0..).map(|index| BlockId { index }).zip(self.blocks.iter())
```

Comment:
Use [`iter::once`](https://doc.rust-lang.org/std/iter/fn.once.html)

Reply:
Where? I don't understand?

Reply:
Me neither, I am waiting for an answer [here](https://github.com/biomejs/biome/pull/28/files#r1315131843) to understand what this function does 😅